src/Test/L1/Tasks/d9bafed4-0b18-4f58-968d-86655b4d2ce9/ps_modules/VstsTaskSdk/OutFunctions.ps1

```diff
@@ -0,0 +1,79 @@
+# TODO: It would be better if the Out-Default function resolved the underlying Out-Default
```

Comment:
We need a better solution for getting these task libraries and tasks than just checking them in. What are our alternatives? Can we get them from the other repos and lock to a specific version? Can we get them as nuget packages?

Reply:
> We need a better solution for getting these task libraries and tasks than just checking them in.

I think I disagree with this - what is the problem you see here?

> Can we get them from the other repos and lock to a specific version?

We don't have built versions of our tasks in any of our repos. We could change that, but that feels heavy to solve this problem

> Can we get them as nuget packages?

Not without auth, which the person running these tests isn't guaranteed to have.